docker-compose.yml

```diff
@@ -137,9 +137,10 @@ services:
       - HYPERSWITCH_SERVER_URL=${HYPERSWITCH_SERVER_URL:-http://hyperswitch-server:8080}
       - HYPERSWITCH_CLIENT_URL=${HYPERSWITCH_CLIENT_URL:-http://localhost:9050}
       - SELF_SERVER_URL=${SELF_SERVER_URL:-http://localhost:5252}
-    depends_on:
-      hyperswitch-server:
-        condition: service_started
+      - SDK_ENV=${SDK_ENV:-local}
+      - ENV_SDK_URL=${ENV_SDK_URL:-http://localhost:9050/HyperLoader.js}
```

Comment:
This won't have the `HyperLoader.js` at the end this is supposed to be the base url only